tinymce-language-selector/plugin.js

```diff
@@ -35,6 +36,13 @@ tinymce.PluginManager.add('language', function(editor) {
 
   const openDialog = (buttonApi) => {
     const selectedNode = editor.selection.getNode();
+    if (['OL', 'UL'].includes(selectedNode.nodeName)) {
```

Comment:
What happened here? Wasn't this one of the features you showed off in the gif?

Reply:
It works if you don't have anything highlighted, but if you do have things highlighted, it gets really hard to properly assign the spans, since you can have only a few items of the list highlighted, the list items can have their own formatting, etc.